- M. Lange _et al._, “CellRank for directed single-cell fate mapping,” _bioRxiv_, p. 2020.10.19.345983, Nov. 2020, doi: [10.1101/2020.10.19.345983](https://doi.org/10.1101/2020.10.19.345983).

- W. Chen _et al._, “A Comprehensive Multi-Center Cross-platform Benchmarking Study of Single-cell RNA Sequencing Using Reference Samples,” _bioRxiv_, p. 2020.03.27.010249, Mar. 2020, doi: [10.1101/2020.03.27.010249](https://doi.org/10.1101/2020.03.27.010249).

- G. La Manno _et al._, “RNA velocity of single cells,” _Nature_, vol. 560, no. 7719, Art. no. 7719, Aug. 2018, doi: [10.1038/s41586-018-0414-6](https://doi.org/10.1038/s41586-018-0414-6).

- A. Haque, J. Engel, S. A. Teichmann, and T. Lönnberg, “A practical guide to single-cell RNA-sequencing for biomedical research and clinical applications,” _Genome Medicine_, vol. 9, no. 1, p. 75, Aug. 2017, doi: [10.1186/s13073-017-0467-4](https://doi.org/10.1186/s13073-017-0467-4).

- D. Arendt _et al._, “The origin and evolution of cell types,” _Nature Reviews Genetics_, vol. 17, no. 12, Art. no. 12, Dec. 2016, doi: [10.1038/nrg.2016.127](https://doi.org/10.1038/nrg.2016.127).

- M. I. Love, W. Huber, and S. Anders, “Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2,” _Genome Biology_, vol. 15, no. 12, p. 550, Dec. 2014, doi: [10.1186/s13059-014-0550-8](https://doi.org/10.1186/s13059-014-0550-8).

- Y. Lazebnik, “Can a biologist fix a radio?—Or, what I learned while studying apoptosis,” _Cancer Cell_, vol. 2, no. 3, pp. 179–182, Sep. 2002, doi: [10.1016/S1535-6108(02)00133-2](https://doi.org/10.1016/S1535-6108(02)00133-2).

- [How UMAP Works](https://umap-learn.readthedocs.io/en/latest/how_umap_works.html) 