## Variance Inference

- R. Lopez, J. Regier, M. B. Cole, M. I. Jordan, and N. Yosef, “Deep generative modeling for single-cell transcriptomics,” _Nature Methods_, vol. 15, no. 12, Art. no. 12, Dec. 2018, doi: [10.1038/s41592-018-0229-2](https://doi.org/10.1038/s41592-018-0229-2).
- V. Svensson, A. Gayoso, N. Yosef, and L. Pachter, “Interpretable factor models of single-cell RNA-seq via variational autoencoders,” _Bioinformatics_, vol. 36, no. 11, pp. 3418–3421, Jun. 2020, doi: [10.1093/bioinformatics/btaa169](https://doi.org/10.1093/bioinformatics/btaa169).
- G. Eraslan, L. M. Simon, M. Mircea, N. S. Mueller, and F. J. Theis, “Single-cell RNA-seq denoising using a deep count autoencoder,” _Nature Communications_, vol. 10, no. 1, Art. no. 1, Jan. 2019, doi: [10.1038/s41467-018-07931-2](https://doi.org/10.1038/s41467-018-07931-2).

## Integration

- T. Stuart _et al._, “Comprehensive Integration of Single-Cell Data,” _Cell_, vol. 177, no. 7, pp. 1888-1902.e21, Jun. 2019, doi: [10.1016/j.cell.2019.05.031](https://doi.org/10.1016/j.cell.2019.05.031).
- I. Korsunsky _et al._, “Fast, sensitive and accurate integration of single-cell data with Harmony,” _Nature Methods_, vol. 16, no. 12, Art. no. 12, Dec. 2019, doi: [10.1038/s41592-019-0619-0](https://doi.org/10.1038/s41592-019-0619-0).
- J. Liu, C. Gao, J. Sodicoff, V. Kozareva, E. Z. Macosko, and J. D. Welch, “Jointly defining cell types from multiple single-cell datasets using LIGER,” _Nature Protocols_, vol. 15, no. 11, Art. no. 11, Nov. 2020, doi: [10.1038/s41596-020-0391-8](https://doi.org/10.1038/s41596-020-0391-8).

## Normalization

- C. Hafemeister and R. Satija, “Normalization and variance stabilization of single-cell RNA-seq data using regularized negative binomial regression,” _Genome Biology_, vol. 20, no. 1, p. 296, Dec. 2019, doi: [10.1186/s13059-019-1874-1](https://doi.org/10.1186/s13059-019-1874-1).
- A. T. L. Lun, K. Bach, and J. C. Marioni, “Pooling across cells to normalize single-cell RNA sequencing data with many zero counts,” _Genome Biol_, vol. 17, no. 1, Art. no. 1, Dec. 2016, doi: [10.1186/s13059-016-0947-7](https://doi.org/10.1186/s13059-016-0947-7).
- W. Tang _et al._, “bayNorm: Bayesian gene expression recovery, imputation and normalization for single-cell RNA-sequencing data,” _Bioinformatics_, vol. 36, no. 4, pp. 1174–1181, Feb. 2020, doi: [10.1093/bioinformatics/btz726](https://doi.org/10.1093/bioinformatics/btz726).
- A. Lun, “Overcoming systematic errors caused by log-transformation of normalized single-cell RNA sequencing data,” _bioRxiv_, p. 404962, Aug. 2018, doi: [10.1101/404962](https://doi.org/10.1101/404962).
- P.-Y. Tung _et al._, “Batch effects and the effective design of single-cell gene expression studies,” _Scientific Reports_, vol. 7, no. 1, Art. no. 1, Jan. 2017, doi: [10.1038/srep39921](https://doi.org/10.1038/srep39921).
- D. Kuang and J. Kim, “Resistant Fit Regression Normalization for Single-cell RNA-seq Data,” in _2020 IEEE 20th International Conference on Bioinformatics and Bioengineering (BIBE)_, Oct. 2020, pp. 236–240, doi: [10.1109/BIBE50027.2020.00046](https://doi.org/10.1109/BIBE50027.2020.00046).
- R. Bacher _et al._, “SCnorm: robust normalization of single-cell RNA-seq data,” _Nature Methods_, vol. 14, no. 6, Art. no. 6, Jun. 2017, doi: [10.1038/nmeth.4263](https://doi.org/10.1038/nmeth.4263).
- J. Wang _et al._, “Data denoising with transfer learning in single-cell transcriptomics,” _Nature Methods_, vol. 16, no. 9, Art. no. 9, Sep. 2019, doi: [10.1038/s41592-019-0537-1](https://doi.org/10.1038/s41592-019-0537-1).