# Single-cell Genomic Reading List

This is a Single-cell Genomic Reading List to take notes on the papers I encountered while learning computational biology. I am a computer science student therefore the list tends to focus on Algorithm and Statistical Inference.

- [10 Must Reads](#10-must-reads)
- [Single-cell RNA Sequencing](#single-cell-rna-sequencing)
  - [Tutorials](#tutorials)
  - [Assembly and Quality Control](#assembly-and-quality-control)
  - [Data Harmonirization](#data-harmonirization)
  - [Dimension Reduction](#dimension-reduction)
  - [Unsuperivsed Clustering](#unsuperivsed-clustering)
  - [Differential Expression Analysis](#differential-expression-analysis)
  - [Enrichment Analysis](#enrichment-analysis)
- [Single-cell Multimodal Analysis](#single-cell-multimodal-analysis)

## 10 Must Reads

- [Haque, A., Engel, J., Teichmann, S.A. et al. A practical guide to single-cell RNA-sequencing for biomedical research and clinical applications. Genome Med 9, 75 (2017)](https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-017-0467-4#citeas)
- [Love, M.I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol 15, 550 (2014).](https://doi.org/10.1186/s13059-014-0550-8)
- [How UMAP Works](https://umap-learn.readthedocs.io/en/latest/how_umap_works.html)

## Single-cell RNA Sequencing Analysis

### Tutorials

- [Current best practices in single‐cell RNA‐seq analysis: a tutorial](https://www.embopress.org/doi/full/10.15252/msb.20188746)
- [Orchestrating Single-Cell Analysis with Bioconductor](https://www.nature.com/articles/s41592-019-0654-x)

### Assembly and Quality Control

- [VERSE: a versatile and efficient RNA-Seq read counting tool (2016)](https://doi.org/10.1101/053306)
- [Ondov, B., Starrett, G., Sappington, A. et al. Mash Screen: high-throughput sequence containment estimation for genome discovery. Genome Biol 20, 232 (2019).](https://doi.org/10.1186/s13059-019-1841-x)

### Data Harmonization

- [Comprehensive integration of single cell data
Tim Stuart, Andrew Butler, Paul Hoffman, Christoph Hafemeister, Efthymia Papalexi, William M. Mauck III, Marlon Stoeckius, Peter Smibert, Rahul Satija (2019).](https://doi.org/10.1016/j.cell.2019.05.031)
- [Liu, J., Gao, C., Sodicoff, J. et al. Jointly defining cell types from multiple single-cemll datasets using LIGER. Nat Protoc (2020).](https://doi.org/10.1038/s41596-020-0391-8)
- [Lopez, R., Regier, J., Cole, M.B. et al. Deep generative modeling for single-cell transcriptomics. Nat Methods 15, 1053–1058 (2018).](https://doi.org/10.1038/s41592-018-0229-2)
- [Korsunsky, I., Millard, N., Fan, J. et al. Fast, sensitive and accurate integration of single-cell data with Harmony. Nat Methods 16, 1289–1296 (2019).](https://doi.org/10.1038/s41592-019-0619-0)
- [Hafemeister, C., Satija, R. Normalization and variance stabilization of single-cell RNA-seq data using regularized negative binomial regression. Genome Biol 20, 296 (2019).](https://doi.org/10.1186/s13059-019-1874-1)
- [L. Lun, A.T., Bach, K. & Marioni, J.C. Pooling across cells to normalize single-cell RNA sequencing data with many zero counts. Genome Biol 17, 75 (2016). ](https://doi.org/10.1186/s13059-016-0947-7)

### Dimension Reduction
- [Everything you did and didn't know about PCA](http://alexhwilliams.info/itsneuronalblog/2016/03/27/pca/)
- [Kobak, D., Berens, P. The art of using t-SNE for single-cell transcriptomics. Nat Commun 10, 5416 (2019).](https://doi.org/10.1038/s41467-019-13056-x)

### Unsupervised Clustering

- [Traag, V.A., Waltman, L. & van Eck, N.J. From Louvain to Leiden: guaranteeing well-connected communities. Sci Rep 9, 5233 (2019).](https://doi.org/10.1038/s41598-019-41695-z)
- [Schwartz, G.W., Zhou, Y., Petrovic, J. et al. TooManyCells identifies and visualizes relationships of single-cell clades. Nat Methods 17, 405–413 (2020).](https://doi.org/10.1038/s41592-020-0748-5)
- [Kiselev, V., Kirschner, K., Schaub, M. et al. SC3: consensus clustering of single-cell RNA-seq data. Nat Methods 14, 483–486 (2017).]( https://doi.org/10.1038/nmeth.4236)

### Differential Expression Analysis

- [Wang, T., Li, B., Nelson, C.E. et al. Comparative analysis of differential gene expression analysis tools for single-cell RNA sequencing data. BMC Bioinformatics 20, 40 (2019).](https://doi.org/10.1186/s12859-019-2599-6)
- [Wilcoxon signed-rank test](http://www.biostathandbook.com/wilcoxonsignedrank.html)

### Enrichment Analysis

- [Bioinformatics enrichment tools: paths toward the comprehensive functional analysis of large gene lists](https://academic.oup.com/nar/article/37/1/1/1026684)
- [Gene set enrichment analysis: A knowledge-based approach for interpreting genome-wide expression profiles](https://doi.org/10.1073/pnas.0506580102)

## Single-cell Multimodal Analysis