## Reviews
- V. Svensson, E. da Veiga Beltrame, and L. Pachter, “A curated database reveals trends in single-cell transcriptomics,” _Database_, vol. 2020, no. baaa073, Jan. 2020, doi: [10.1093/database/baaa073](https://doi.org/10.1093/database/baaa073).
- V. Svensson, R. Vento-Tormo, and S. A. Teichmann, “Exponential scaling of single-cell RNA-seq in the past decade,” _Nature Protocols_, vol. 13, no. 4, Art. no. 4, Apr. 2018, doi: [10.1038/nprot.2017.149](https://doi.org/10.1038/nprot.2017.149).

## Experiments
- B. Pijuan-Sala _et al._, “A single-cell molecular map of mouse gastrulation and early organogenesis,” _Nature_, vol. 566, no. 7745, Art. no. 7745, Feb. 2019, doi: [10.1038/s41586-019-0933-9](https://doi.org/10.1038/s41586-019-0933-9).

- A. Zeisel _et al._, “Cell types in the mouse cortex and hippocampus revealed by single-cell RNA-seq,” _Science_, vol. 347, no. 6226, pp. 1138–1142, Mar. 2015, doi: [10.1126/science.aaa1934](https://doi.org/10.1126/science.aaa1934).

- A. M. Klein _et al._, “Droplet Barcoding for Single-Cell Transcriptomics Applied to Embryonic Stem Cells,” _Cell_, vol. 161, no. 5, pp. 1187–1201, May 2015, doi: [10.1016/j.cell.2015.04.044](https://doi.org/10.1016/j.cell.2015.04.044).

- M. Saint _et al._, “Single-cell imaging and RNA sequencing reveal patterns of gene expression heterogeneity during fission yeast growth and adaptation,” _Nature Microbiology_, vol. 4, no. 3, Art. no. 3, Mar. 2019, doi: [10.1038/s41564-018-0330-4](https://doi.org/10.1038/s41564-018-0330-4).